### ISSUE TITLE
Is the trained model released for test?

### ISSUE THREAD

**Issue by popoala** [Flash-321/ARLDM#2]
Wonderful job! Is the model released ? Hope to see what does it lead to.

**Comment by xichenpan** [comment]
Yes, we have this plan, and we will prepare a huggingface space to host the checkpoint. The release is currently under internal review of Alibaba Group. After that, we will host it in huggingface ASAP.

**Comment by MeixiChenTracy** [comment]
> Yes, we have this plan, and we will prepare a huggingface space to host the checkpoint. The release is currently under internal review of Alibaba Group. After that, we will host it in huggingface ASAP.

@Flash-321 Hi, thank you so much for this wonderful work! I wonder if you would be able provide a rough estimate of when the review process is likely to finish? I am really looking forward to the trained checkpoint and the huggingface space. Knowing roughly when they will be released will be very helpful for scheduling my work.
Many thanks.

**Comment by xichenpan** [comment]
> > Yes, we have this plan, and we will prepare a huggingface space to host the checkpoint. The release is currently under internal review of Alibaba Group. After that, we will host it in huggingface ASAP.
> 
> @Flash-321 Hi, thank you so much for this wonderful work! I wonder if you would be able provide a rough estimate of when the review process is likely to finish? I am really looking forward to the trained checkpoint and the huggingface space. Knowing roughly when they will be released will be very helpful for scheduling my work. Many thanks.

Hi, thanks a lot for your attention. My sincere apologies for the late ckpt release. Last few weeks, Alibaba is undergoing a covid pandemic, and most of my colleges in charge of this thread caught a fever. Now it is quite close to Chinese new year, and I am regret that I cannot expect the time for ckpt release. btw, I also checked out from Alibaba, but I would inform my mentor about this situation and ask him to push forward the disclosure review process.

**Comment by xichenpan** [comment]
> > > Yes, we have this plan, and we will prepare a huggingface space to host the checkpoint. The release is currently under internal review of Alibaba Group. After that, we will host it in huggingface ASAP.
> > 
> > 
> > @Flash-321 Hi, thank you so much for this wonderful work! I wonder if you would be able provide a rough estimate of when the review process is likely to finish? I am really looking forward to the trained checkpoint and the huggingface space. Knowing roughly when they will be released will be very helpful for scheduling my work. Many thanks.
> 
> Hi, thanks a lot for your attention. My sincere apologies for the late ckpt release. Last few weeks, Alibaba is undergoing a covid pandemic, and most of my colleges in charge of this thread caught a fever. Now it is quite close to Chinese new year, and I am regret that I cannot expect the time for ckpt release. btw, I also checked out from Alibaba, but I would inform my mentor about this situation and ask him to push forward the disclosure review process.

btw, the gradio code prepared for demo is ready, I will host a space in huggingface for the demo asap after the review finished

**Comment by MeixiChenTracy** [comment]
> > > > Yes, we have this plan, and we will prepare a huggingface space to host the checkpoint. The release is currently under internal review of Alibaba Group. After that, we will host it in huggingface ASAP.
> > > 
> > > 
> > > @Flash-321 Hi, thank you so much for this wonderful work! I wonder if you would be able provide a rough estimate of when the review process is likely to finish? I am really looking forward to the trained checkpoint and the huggingface space. Knowing roughly when they will be released will be very helpful for scheduling my work. Many thanks.
> > 
> > 
> > Hi, thanks a lot for your attention. My sincere apologies for the late ckpt release. Last few weeks, Alibaba is undergoing a covid pandemic, and most of my colleges in charge of this thread caught a fever. Now it is quite close to Chinese new year, and I am regret that I cannot expect the time for ckpt release. btw, I also checked out from Alibaba, but I would inform my mentor about this situation and ask him to push forward the disclosure review process.
> 
> btw, the gradio code prepared for demo is ready, I will host a space in huggingface for the demo asap after the review finished

@Flash-321 Many thanks for your reply! Wish your colleges get well soon, and Happy Chinese New Year!

**Comment by xichenpan** [comment]
> > > > > Yes, we have this plan, and we will prepare a huggingface space to host the checkpoint. The release is currently under internal review of Alibaba Group. After that, we will host it in huggingface ASAP.
> > > > 
> > > > 
> > > > @Flash-321 Hi, thank you so much for this wonderful work! I wonder if you would be able provide a rough estimate of when the review process is likely to finish? I am really looking forward to the trained checkpoint and the huggingface space. Knowing roughly when they will be released will be very helpful for scheduling my work. Many thanks.
> > > 
> > > 
> > > Hi, thanks a lot for your attention. My sincere apologies for the late ckpt release. Last few weeks, Alibaba is undergoing a covid pandemic, and most of my colleges in charge of this thread caught a fever. Now it is quite close to Chinese new year, and I am regret that I cannot expect the time for ckpt release. btw, I also checked out from Alibaba, but I would inform my mentor about this situation and ask him to push forward the disclosure review process.
> > 
> > 
> > btw, the gradio code prepared for demo is ready, I will host a space in huggingface for the demo asap after the review finished
> 
> @Flash-321 Many thanks for your reply! Wish your colleges get well soon, and Happy Chinese New Year!

@MeixiChenTracy Thanks for your understanding. Happy Chinese New Year!

**Comment by yangqing-20** [comment]
Hi,sorry for bothering. I am wondering the same question. : )

**Comment by CIntellifusion** [comment]
Sorry for bothering. I am looking forward to seeing your fantastic this model

**Comment by K-h-Huang** [comment]
你好！我对这个项目很感谢趣，请问现在已经在huggingface发布了相关demo吗？

**Comment by S-Manasa** [comment]
Hi @xichenpan , I was wondering the same thing, Is the pre-trained model ready or huggingFace demo ready?

**Comment by b67294** [comment]
wonder too, is there any ckpt could use?

**Comment by tearknows** [comment]
佬，有没有放到huggingface呀，毕设选了这个方向的😢
